CONFIG.md

```diff
@@ -327,6 +327,29 @@ This is an example of the settings for the validate tasks:
 }
 ```
 
+**eslint**: To enable eslint, set to `true` to use default options or an object
+with the following optional settings.
+
+**eslint.configFile**: The path to the eslint config file to use. If no value
+is specified, then `.eslintrc` in the project root is used.
+
+**eslint.ignoreError**: Set to `false` to trigger a Grunt error if validation
+fails, which will prevent subsequent tasks from executing.
```

Comment:
I think the bigger deal is to fail builds on JS coding style errors.

Also, since we have a similar setting for phpcs, should we change this to a general validate.syntax.ignoreError setting? The granularity is nice but I can't really picture a case where I'd want to toggle different levels of strictness for different types of code.